addon/types/table.js

```diff
@@ -44,6 +46,7 @@ export default EmberObject.extend({
       column.setProperties({
         orderBy: column.orderBy || null,
         orderKey: column.orderKey || column.key,
+        size: COLUMN_SIZES[Math.floor(Math.random() * 3)] || 'm',
```

Comment:
remove the constant and use `size: column.size || 'm'`. will use the random on the callee app if needed (until the backend supports it)